modules/atomic-widgets/validators/styles-validator.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\Validators;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Styles_Validator {
+
+	private array $schema;
+
+	public function __construct( array $schema ) {
+		$this->schema = $schema;
+	}
+
+	public static function make( array $schema ): self {
+		return new static( $schema );
+	}
+
+	/**
+	 * @param array $styles
+	 * The key of each item represents the style id,
+	 * and the value is the style object to validate
+	 *
+	 * @return array{
+	 *     0: bool,
+	 *     1: array<string, mixed>,
+	 *     2: array<string>
+	 * }
+	 */
+	public function validate( array $styles ): array {
+		$errors = [];
+
+		foreach ( $styles as $style_id => $style ) {
+			try {
+				$this->validate_id( $style['id'] );
```

Comment:
```suggestion
				$this->validate_id( $style['id'] );
```

isset? maybe just check that this is exists and it is a string